testing/regression/tests/dataGrid/keyboardNavigation.ts

```diff
@@ -0,0 +1,31 @@
+import { Selector } from 'testcafe';
+import { gridInit, getTestFileUrl } from '../../helpers/testHelper';
+
+fixture `Keyboard Navigation`
+    .page(getTestFileUrl("tests/dataGrid/container.html"));
+
+test("Cell should not highlighted after editing another cell when startEditAction is 'dblClick'", async t => {
+    await t
+        .doubleClick(Selector(".dx-datagrid .dx-data-row:nth-child(2) td:nth-child(2)"))
+        .wait(500)
+        .click(Selector(".dx-datagrid .dx-data-row:nth-child(1) td:nth-child(2)"))
+        .wait(500)
+        .doubleClick(Selector(".dx-datagrid .dx-data-row:nth-child(2) td:nth-child(2)"))
+        .wait(500)
+        .click(Selector(".dx-datagrid .dx-data-row:nth-child(1) td:nth-child(2)"))
```

Comment:
It will be better to use [pageModel](https://devexpress.github.io/testcafe/documentation/recipes/extract-reusable-test-code/use-page-model.html) for element describing